Sources/MuxUploadSDK/Internal Utilities/ChunkedFile.swift

```diff
@@ -28,7 +28,7 @@ class ChunkedFile {
     
     /// Reads the next chunk from the file, advancing the file for the next read
     ///  This method does synchronous I/O, so call it in the background
-    public func readNextChunk() -> Result<FileChunk, Error> {
+    func readNextChunk() -> Result<FileChunk, Error> {
```

Comment:
I'm surprised the compiler didn't complain about this being `public` despite `ChunkedFile` remaining internal

Reply:
I would have at least expected a warning

Reply:
![image](https://github.com/muxinc/swift-upload-sdk/assets/102617203/efd27262-8da5-468b-bc5e-851099e76872)